### DISCUSSION TITLE
Hot reloading with changes to non-python files

### DISCUSSION THREAD

**Discussion by rohitsathish** [reflex-dev/reflex#2243] [Q&A]
I'm loading an external stylesheet to my reflex app and making all styling changes in a style.css. 
However, I'm forced to make changes in .py files to trigger the hot reloading. Is there any way to include other types of files in hot reloading?

**Comment by picklelo** [comment]
Currently we only look at the `.py` files for hot reloads as we use the Uvicorn reload feature: https://www.uvicorn.org/settings/

It would be nice to support other file types for use cases like you mentioned. I created an issue #2248 for us to implement this.
